### PR TITLE
Routing, Styling and Testing - React Calculator

### PR DESCRIPTION
**Pull request by RyelBanfield**
This PR contains implementing the react routing features, some additional styling of the project and testing of the components and logic.

### CODE REVIEW

File: src/__tests__/Calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import Calculate from '../logic/calculate';
+
+test('should calculate the answer', () => {
```
Comment (trekab) [inline src/__tests__/Calculate.test.js:3]: All your tests have this same descriptive message. :thinking: I think it is better to make them more descriptive by referring to the calculation/test they perform e.g. `should calculate the sum`, `should calculate the division`, e.t.c

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @RyelBanfield! Your work on the test suite is commendable so far since you've set it up well and tested some functionality. :+1: However, there are some changes you need to work on to have your testing(PR) approved. These include:

- [ ] At a minimum, you should have snapshot tests for `Home`, `Calculator`, and `Quote`.
- [ ] Check out the code comment below about your tests' descriptive messages.

_Please make the necessary changes and submit another code review request._
__[Treasure Kabareebe](https://trekab.github.io)__

**Comment by RyelBanfield** [comment]
@trekab thank you for your review mate. Can I ask why the project specs say use snapshots for just the quote page? It's not clear and should be updated if that's the case.
image

**Comment by trekab** [comment]
Is your question in regard to how small the quote page is? If so, I think the main focus for this project is to help you get enough practice writing tests for React projects(especially making use of the snapshot testing). It is an area easily ingored by new developers.

**Comment by rukundoeric** [comment]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯 
  - Issues raised from the previous review are all addressed. 💯 
  

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/),
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**